cpp/include/cudf/utilities/traits.hpp

```diff
@@ -102,6 +104,8 @@ using is_timestamp_t = cuda::std::disjunction<std::is_same<cudf::timestamp_D, T>
 /// Checks if a type is a duration type.
 template <typename T>
 using is_duration_t = cuda::std::disjunction<std::is_same<cudf::duration_D, T>,
+                                             std::is_same<cudf::duration_h, T>,
+                                             std::is_same<cudf::duration_m, T>,
```

Comment:
*nit*: These additions now seem unused? What are they needed for?

Reply:
I believe they were just missing and should've always been here but not noticed until now. 
They are not needed for this PR.

Reply:
David's right. I'll can try to add to add them in a future PR, but will delete them now as they are not needed in this PR.

Reply:
I don't mind them being here ,and they need to come in somewhere so let's leave them be :)